docs/sample_config.yaml

```diff
@@ -1137,6 +1137,11 @@ account_threepid_delegates:
     #email: https://example.com     # Delegate email sending to example.com
     #msisdn: http://localhost:8090  # Delegate SMS sending to this local process
 
+    # Only delegate threepid registration, not password resets. Defaults to
+    # false.
+    #
+    #registration_only: true
```

Comment:
I'm a bit concerned about how this will work if we need to add any future knobs, e.g. if we end up with `password_resets_only` too, which could be quite confusing. Maybe we should have something like `delegate_for: [registration]` or something?

Reply:
That was my plan initially, but the logic around this config section is actually quite complicated, and changing it that way would have required quite a bit of plumbing across a big chunk of code. Which is why I went for this solution instead.